package.json

```diff
@@ -26,7 +26,8 @@
   "dependencies": {
     "chalk": "^3.0.0",
     "cosmiconfig": "^6.0.0",
-    "create-jest-runner": "^0.6.0"
+    "create-jest-runner": "^0.6.0",
+    "dot-prop": "^5.3.0"
```

Comment:
This author is planning to make all their packages ESM-only - I’d strongly suggest avoiding adding them as a dependency.

Reply:
I went for v5 since v6 doesn't support node 8. We can migrate to some other module later if needed. Unless you have a good suggestion for a module that's not this or lodash? 😀

Reply:
not sure - the `ESLint` class requires a bunch of stuff in `configOverrides` instead of in the root object, and deep merging is quite hellish in JS (without some module). Going with a dotted setter seemed like a nice shortcut for the specific CLI options we've added support for

(I'm not entirely sure why we have this normalization instead of passing through raw as-is, but that's a separate discussion)

Reply:
I'll land this, and if we figure out some cleaner way of dealing with this we can revisit. It's not exposed, so it's an implementation detail we can change at will